lib/eco/cflexer/lexer.py

```diff
@@ -168,15 +170,18 @@ def find_next_token(self):
                 assert stop >= 0
                 if start == stop:
                     source = self.text[start:]
-                    result = self.make_token(start, self.last_matched_state, source, lookahead = i - stop)
+                    raise LexingError("Could not successfully parse %s" % source)
+                    result = self.make_token(start, -1, self.last_matched_state, lookahead = i - stop)
```

Comment:
The rest of this `if`s block almost certainly can't be executed (because of the `raise`).